software/xx-network-haven.yml

```diff
@@ -0,0 +1,11 @@
+name: "Haven"
+website_url: "https://xx.network"
+source_code_url: "https://git.xx.network/elixxir/speakeasy-web/"
+description: "Anonymous, distributed, mixnet-based text messaging chat app. Uses xx network's cMixx technology by Dr. David Chaum. Not to be confused with Haven the blogging system."
```

Comment:
I do not understand what this does or how it works. The description should state this as plainly as possible.

> https://xx.network
>The First High-Speed Quantum Resistant Decentralized Mixnet with Blockchain.
>Protect metadata for dApps, chains, and businesses with advanced quantum-resistant security.
>Precomputation with cMixx delivers real-time calculations hundreds of times faster than other mixnets.

Yeah, this does not help...

> https://git.xx.network/elixxir/speakeasy-web#getting-started
> run the development server:

Are there installation instructions for production use?